AUTHORS

```diff
@@ -68,6 +68,7 @@ Arpit Bandejiya <arpitbandejiya@gmail.com>
 Arun Kumar <arunkumarm1995@gmail.com>
 Arunabh Ghosh <arunabhghosh98@gmail.com>
 Aryaman Gupta <aryaman.gupta.met19@iitbhu.ac.in>
+Aryaman Gupta <aryaman.gupta.met19@itbhu.ac.in>
```

Comment:
Somehow the script added a duplicate item here, looks like the email is different!

Reply:
Removed!